flink/README.md

```diff
@@ -208,3 +210,25 @@ Received a total of 1 row
 | tikv.sink.deduplicate                               | false                                                                          | Only work when sink option is `TIKV`. If value is `true`, duplicate row will be de-duplicated. If `false`, you should make sure each row is unique otherwise exception will be thrown.                                                                                                                                                                                                                                                                    |
 
 
+## TableFactory(deprecated)
+
+Attention: TableFactory is deprecated, only support before Flink 1.13(included).
+
+TiBigData also implements the Flink TableFactory API, but we don't recommend you to use it, it will introduce difficulties related to data type conversion and column alignment, which will increase the cost of using it. We stop supporting it in Flink-1.14, so this section is only a brief introduction.
```

Comment:
We stop supporting it in Flink-1.13+.

Reply:
Due to back-compatibility, we still need to support it in Flink-1.13